Development environment/Laravel Homestead setup.md

```diff
@@ -0,0 +1,147 @@
+Homestead is an official, pre-packaged Vagrant box from Laravel team. Its default approach is having multiple sites set up on a single Vagrant instance, which makes adding new sites a trivial task. For more advanced requirements there is an option to set up each site on its own instance.
+
+Homestead runs on any Windows, Mac, or Linux system, and includes the Nginx web server, PHP 7.3, PHP 7.2, PHP 7.1, MySQL, PostgreSQL, Redis, Memcached, Node, etc.. For full list of features check out the [offical documentation](https://laravel.com/docs/5.8/homestead)
+
+## Installing Laravel Homestead
+
+As any other Vagrant boxes, Homestead requires a few prerequisites installed:
+
+* [Vagrant](https://www.vagrantup.com/downloads.html)
+* [Git](https://git-scm.com/downloads)
+* Either one of the following: [VirtualBox](https://www.virtualbox.org/wiki/Downloads), [VMWare](https://my.vmware.com/en/web/vmware/downloads)<sup>1</sup>, [Parallels](https://www.parallels.com/products/desktop/)<sup>2</sup>, or [Hyper-V](https://docs.microsoft.com/en-us/virtualization/hyper-v-on-windows/quick-start/enable-hyper-v)
+
+<sup>1 Requires the purchase of both VMware Fusion / Workstation and the VMware Vagrant plug-in.</sup>
+
+<sup>2 Not free, but has a free trial period and requires a free Vagrant plug-in</sup>
+
+For easiest setup use [VirtualBox](https://www.virtualbox.org/wiki/Downloads) as it is free, multi-platform and regularly updated.
+
+After everything is installed we can begin with Homestead setup.
+
+First add/download the `laravel/homestead` box to your Vagrant installation from terminal:
+
+```sh
+vagrant box add laravel/homestead
+```
+
+Clone the Homestead repository in a directory that will hold all your Homestead projects:
+
+```sh
+git clone https://github.com/laravel/homestead.git ~/homestead
+```
+
+Go into the directory that repo was cloned in and checkout the latest stable release tag:
+
+```sh
+cd ~/homestead
+
+# Checkout 8.4.0, or a newer stable tag.
+git checkout v8.4.0
```

Comment:
can we remove this version number here but just add a link to the repo where they can check the latest version and just put this "git checkout v8.4.0" as an example so we don't need to change it every time there is a new version?

Reply:
👍